texoid/server.py

```diff
@@ -91,7 +92,7 @@ def handle_request(self):
                              'height': height}
                 }
         except Exception as error:
-            print error.message
+            logging.error(error.message)
```

Comment:
You should create a logger, not use the global logging. You should also use exception instead of error.